### PR TITLE
Add Big Town Extensions

### PR DESCRIPTION
**Pull request by Superfly-Johnson**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -2454,6 +2454,26 @@ plugins:
       - crc: 0xEBAC2B12
         util: 'FNVEdit v4.0.4b'
 
+  - name: 'Big Town Extensions.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/74825/' ]
```
Comment (pStyl3) [inline masterlist.yaml]: Please add the link name to the url.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Review by pStyl3 — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -2454,6 +2454,34 @@ plugins:
...
 
+  - name: 'Big Town Extensions.esm'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/74825/'
+        name: 'Big Town Extensions'
+    req:
+      - *JIPLN
+      - *johnnyGuitar
+      - *TTWv320
+    after: ['AWorldOfPainFO3.esm']
```
Comment (pStyl3) [inline masterlist.yaml]: Please add empty spaces like this: `[ 'AWorldOfPainFO3.esm' ]`

Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    after: [ 'AWorldOfPainFO3.esm' ]
```

File: masterlist.yaml [hunk 3]
```diff
@@ -2454,6 +2454,34 @@ plugins:
...
+          - '[TTW Reputations - Big Town Extensions](https://www.nexusmods.com/newvegas/mods/83169/)'
+        condition: 'active("TTW Reputations.esm") and not active("TTW Reputations - Big Town Extensions.esp")'
+      - <<: *wildEdits
+        subs: [ 'Delete **0602D934** & **06058B9E** from **Big Town Extensions.esm**.' ]
+        condition: 'checksum("Big Town Extensions.esm", A99EFA1E) or checksum("Big Town Extensions.esm", 39F84E9C)'
+    tag:
+      - C.Owner
+      - Names
+    dirty:
+      # 5.01
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**